### PR TITLE
feat: generate a random password at user creation if not defined

### PR DESCRIPTION
**Pull request by Filirom1**
Hello,

I would like to create users, but I don't want to manage password in terraform.

I create this Pull Request to automatically generate a secured password that the user will not know. To log in, the user will need to click on "forget the password?" to generate a new one.

What do you think about this feature ?
Cheers
Romain

### CODE REVIEW

File: gsuite/resource_user.go [hunk 1]
```diff
@@ -487,6 +488,15 @@ func resourceUserCreate(d *schema.ResourceData, meta interface{}) error {
...
 	}
 
+  if user.Password == "" {
+    res, err := password.Generate(32, 4, 4, false, false)
+    if err != nil {
+      return err
+    }
+    user.Password = res
+		log.Printf("[INFO] A random password was generated for the user")
+  }
```
Comment (DeviaVir) [inline gsuite/resource_user.go:498]: Can you run a `go fmt` on this file?

Comment (Filirom1) [reply gsuite/resource_user.go]: Sorry about it, it's done now

**Comment by DeviaVir** [comment]
Thank you for contributing!

**Comment by Filirom1** [comment]
Thanks!

**Pull request merged**